src/sentry/lang/java/plugin.py

```diff
@@ -317,7 +317,10 @@ def exceptions_differ(a, b):
             symbolicator_stacktraces, self.stacktrace_infos
         ):
             # NOTE: lets hope that `stacktrace_info` has the already processed frames
-            python_stacktrace = stacktrace_info.container.get("stacktrace")
+            python_stacktrace = get_path(stacktrace_info, "container", "stacktrace", default=None)
```

Comment:
I think you should check `stacktrace_info.container` specifically. `get_path` probably won’t work on `.` properties?